addons/web/static/src/core/model_field_selector/model_field_selector_popover.js

```diff
@@ -107,5 +107,6 @@ Object.assign(ModelFieldSelectorPopover, {
         loadChain: Function,
         filter: Function,
         close: Function,
+        followRelations: Boolean,
```

Comment:
the default is true in the ModelFieldSelector and false in this component, isn't it confusing? Is there a reason for this?

Reply:
I don't think this component is supposed to be used elsewhere than in ModelFieldSelector
none of the other props are optional or have defaults and they could but it doesn't cost anything to do it for this one